front-end/src/app/candidate/data-candidate/data-candidate.component.ts

```diff
@@ -179,4 +183,14 @@ export class DataCandidateComponent implements OnInit {
       this.imageBinary = null;
```

Comment:
I had to remove this line to be able to run the project

Reply:
> I had to remove this line to be able to run the project

No problem